sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -9,13 +9,30 @@ import androidx.fragment.app.FragmentManager.FragmentLifecycleCallbacks
 import io.sentry.Breadcrumb
 import io.sentry.HubAdapter
 import io.sentry.IHub
+import io.sentry.ISpan
 import io.sentry.SentryLevel.INFO
+import io.sentry.SpanStatus
 
 @Suppress("TooManyFunctions")
 class SentryFragmentLifecycleCallbacks(
-    private val hub: IHub = HubAdapter.getInstance()
+    private val hub: IHub = HubAdapter.getInstance(),
+    val enableFragmentLifecycleBreadcrumbs: Boolean,
+    val enableAutoFragmentLifecycleTracing: Boolean
 ) : FragmentLifecycleCallbacks() {
 
+    constructor(
+        enableFragmentLifecycleBreadcrumbs: Boolean = true,
+        enableAutoFragmentLifecycleTracing: Boolean = false
+    ) : this(
+        hub = HubAdapter.getInstance(),
+        enableFragmentLifecycleBreadcrumbs = enableFragmentLifecycleBreadcrumbs,
+        enableAutoFragmentLifecycleTracing = enableAutoFragmentLifecycleTracing
+    )
+
+    private val isPerformanceEnabled = hub.options.isTracingEnabled && enableAutoFragmentLifecycleTracing
```

Comment:
Good catch @bruno-garcia. I think this would to the trick
```suggestion
    private val isPerformanceEnabled get() = hub.options.isTracingEnabled && enableAutoFragmentLifecycleTracing
```

Reply:
its a new instance per activity, as the callback is bound to the fragment manager, and each activity has its own fragmentmanager, I guess its the same as https://github.com/getsentry/sentry-java/pull/1528#discussion_r658519308
so in the case that a new activity is setup and `isTracingEnabled` is disabled, nothing is gonna happen.